inputs/emission_constraints/state_cap.csv

```diff
@@ -1,206 +1,206 @@
-WI,2032,18135764.14
-WI,2033,17128221.69
-WI,2034,16120679.24
-WI,2035,15113136.78
-WI,2036,14105594.33
-WI,2037,13098051.88
-WI,2038,12090509.43
-WI,2039,11082966.97
-WI,2040,10075424.52
-WI,2041,9067882.07
-WI,2042,8060339.618
-WI,2043,7052797.166
-WI,2044,6045254.714
-WI,2045,5037712.261
-WI,2046,4030169.809
-WI,2047,3022627.357
-WI,2048,2015084.905
-WI,2049,1007542.452
-WI,2050,0
+*st,t,metrictonCO2
+CA,2010,100000000
+CA,2011,100000000
+CA,2012,64759065.49
+CA,2013,64139680.61
+CA,2014,77396623.1
+CA,2015,69495820
+CA,2016,67396098.67
+CA,2017,65296377.33
+CA,2018,63196656
+CA,2019,61096934.67
+CA,2020,58997213.33
+CA,2021,56897492
+CA,2022,56897492
+CA,2023,55000000
+CA,2024,53000000
+CA,2025,50000000
+CA,2026,48000000
+CA,2027,45000000
+CA,2028,43000000
+CA,2029,40000000
+CA,2030,38000000
+CA,2031,36000000
+CA,2032,35000000
+CA,2033,33000000
+CA,2034,32000000
+CA,2035,30000000
+CA,2036,29000000
+CA,2037,27000000
+CA,2038,26000000
+CA,2039,24000000
+CA,2040,23000000
+CA,2041,21000000
+CA,2042,20000000
+CA,2043,18000000
+CA,2044,17000000
+CA,2045,15000000
+CA,2046,15000000
+CA,2047,15000000
+CA,2048,15000000
+CA,2049,15000000
+CA,2050,15000000
+DE,2010,10000000
+DE,2011,10000000
+DE,2012,10000000
+DE,2013,10000000
+DE,2014,10000000
+DE,2015,10000000
+DE,2016,10000000
+DE,2017,10000000
+DE,2018,10000000
+DE,2019,10000000
+DE,2020,10000000
+DE,2021,1800000
+DE,2022,1700000
+DE,2023,1600000
+DE,2024,1500000
+DE,2025,1400000
+DE,2026,1300000
+DE,2027,1200000
+DE,2028,1100000
+DE,2029,1000000
+DE,2030,900000
+DE,2031,855000
+DE,2032,810000
+DE,2033,765000
+DE,2034,720000
+DE,2035,675000
+DE,2036,630000
+DE,2037,585000
+DE,2038,540000
+DE,2039,495000
+DE,2040,450000
+DE,2041,405000
+DE,2042,360000
+DE,2043,315000
+DE,2044,270000
+DE,2045,225000
+DE,2046,180000
+DE,2047,135000
+DE,2048,90000
+DE,2049,45000
+DE,2050,0
+LA,2010,100000000
+LA,2011,100000000
+LA,2012,100000000
+LA,2013,100000000
+LA,2014,100000000
+LA,2015,100000000
+LA,2016,100000000
+LA,2017,100000000
+LA,2018,100000000
+LA,2019,100000000
+LA,2020,100000000
+LA,2021,100000000
+LA,2022,100000000
+LA,2023,100000000
+LA,2024,100000000
+LA,2025,100000000
+LA,2026,32875013.39
+LA,2027,30776608.28
+LA,2028,28678203.17
+LA,2029,26579798.06
+LA,2030,24481392.95
+LA,2031,22382987.84
+LA,2032,20284582.73
+LA,2033,18186177.62
+LA,2034,16087772.51
+LA,2035,13989367.4
+LA,2036,12940164.84
+LA,2037,11890962.29
+LA,2038,10841759.73
+LA,2039,9792557.179
+LA,2040,8743354.624
+LA,2041,8043886.254
+LA,2042,7344417.884
+LA,2043,6644949.514
+LA,2044,5945481.144
+LA,2045,5246012.774
+LA,2046,4196810.22
+LA,2047,3147607.665
+LA,2048,2098405.11
+LA,2049,1049202.555
+LA,2050,0
+MI,2010,100000000
+MI,2011,100000000
+MI,2012,100000000
+MI,2013,100000000
+MI,2014,100000000
+MI,2015,100000000
+MI,2016,100000000
+MI,2017,100000000
+MI,2018,100000000
+MI,2019,100000000
+MI,2020,100000000
+MI,2021,100000000
+MI,2022,100000000
+MI,2023,100000000
+MI,2024,100000000
+MI,2025,100000000
+MI,2026,47554062.92

```

Comment:
I think we should start the emissions cap in 2023 based on the values I sent. For MI, the 2023 value should be 50449297.97